doc/api/cli.md

```diff
@@ -1116,6 +1116,26 @@ unless either the `--pending-deprecation` command-line flag, or the
 are used to provide a kind of selective "early warning" mechanism that
 developers may leverage to detect deprecated API usage.
 
+### `--permission-fs-path-delimiter`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+This flag configures the file system path delimiter for permissions using
+the [Permission Model][].
+
+Examples can be found in the [File System Permissions][] documentation.
+
+Special shell characters such as `;` must be escaped or quoted:
+
+```bash
+node --experimental-permission --permission-fs-path-delimiter=\; \
+--allow-fs-read=/path/to/index.js index.js
```

Comment:
I think the example should contain the `;` to clarify. 

Also, if you call `--permission-fs-path-delimiter=";"` it should solve the escape part.

Reply:
Yes it does. I commented both options were possible

Reply:
I'd leave this warning to `Accessing files with comma in path` part and use ; here directly.

Reply:
I think I did what you meant, can you confirm?